docx/oxml/section.py

```diff
@@ -42,12 +44,16 @@ class CT_SectPr(BaseOxmlElement):
     ``<w:sectPr>`` element, the container element for section properties.
     """
     __child_sequence__ = (
-        'w:footnotePr', 'w:endnotePr', 'w:type', 'w:pgSz', 'w:pgMar',
-        'w:paperSrc', 'w:pgBorders', 'w:lnNumType', 'w:pgNumType', 'w:cols',
-        'w:formProt', 'w:vAlign', 'w:noEndnote', 'w:titlePg',
+        'w:headerReference', 'w:footnotePr', 'w:endnotePr', 'w:type', 'w:pgSz',
+        'w:pgMar', 'w:paperSrc', 'w:pgBorders', 'w:lnNumType', 'w:pgNumType',
+        'w:cols', 'w:formProt', 'w:vAlign', 'w:noEndnote', 'w:titlePg',
```

Comment:
* the convention for these changed to _tag_seq = (..., you'll see it around in the oxml layer. Also you see on those a `del _tag_seq` statement after the element declarations. If you have a look at a few of those you'll see how to tidy this one up.
* The child tag names should appear in exactly the same sequence as they do in the XML schema for this element; they're used for sequencing new tags in the right order.